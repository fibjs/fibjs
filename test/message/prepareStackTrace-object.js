'use strict';

function prepareStackTrace(error, stack) {
  return new Error("custom Error Object!");
}
Error.prepareStackTrace = prepareStackTrace;

throw new Error();
