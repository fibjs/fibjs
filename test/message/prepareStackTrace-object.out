*test*message*prepareStackTrace-object.js:*
throw new Error();
^
Error: custom Error Object!
